custom-namespaces.yaml

```diff
@@ -409,6 +414,11 @@ firefox_accounts:
         - channel: release
           table: mozdata.firefox_accounts.fxa_log_device_command_events
   explores:
+    fxa_all_events:
+      type: events_explore
+      views:
+        base_view: events
```

Comment:
This will base it on `fxa_content_auth_oauth_events` (per the `events` view definition above), which isn't what we want.